API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/notifications/notifications_comment_reply.json

```diff
@@ -0,0 +1,78 @@
+{
+  "scenarioName": "comment_flow",
+	"requiredScenarioState": "initial list",
+	"newScenarioState": "replied",
+  "request": {
```

Comment:
i'm not sure why it's showing an indentation issue here, on my local this looks fine 🤔 

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/17252150/34fc077c-88ba-4075-a790-783f52c678ae)

Reply:
<details><summary>These two lines are using Tab instead of 2 spaces.</summary>
<img width="300" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/42008628/5cff7f36-e9ef-436a-b86c-ed90a6a4e7ad">
</details> 

<details><summary>I know you use VS Code and you probably already know about it, but I have `Render Whitespace` always enabled and it helps me a lot.</summary>
<img width="300" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/42008628/cf745ef8-5ccb-4bb3-be27-8e83b826accb">
</details>

Reply:
Thanks for sharing this! I actually don't know about the `Render Whitespace` setting 😬 I'll start using that so I don't see "mysteries" like this anymore on my end.